libvips/meson.build

```diff
@@ -0,0 +1,126 @@
+libvips_sources = []
+libvips_components = []
+subdir('include/vips')
+subdir('foreign')
+if get_option('deprecated')
+    subdir('deprecated')
+endif
+subdir('arithmetic')
+subdir('resample')
+subdir('colour')
+subdir('conversion')
+subdir('convolution')
+subdir('freqfilt')
+subdir('histogram')
+subdir('draw')
+subdir('iofuncs')
+subdir('morphology')
+subdir('mosaicing')
+subdir('create')
+
+libvips_lib = library('vips',
+    enumtypes,
+    link_whole: libvips_components,
+    dependencies: libvips_deps,
+    version: library_version,
+    install: true,
+)
+
+libvips_dep = declare_dependency(
+    link_with: libvips_lib,
+    dependencies: libvips_deps,
+    sources: [ enumtypes, soname_header ]
+)
+
+pkg.generate(
+    libvips_lib,
+    requires: [ glib_dep, gio_dep, gobject_dep ],
+    name: 'vips',
+    description: 'Image processing library',
+)
+
+if get_option('introspection')
+    vips_gir = gnome.generate_gir(
```

Comment:
fixed! thanks!

Reply:
Actually, now I try it, the `--quiet` option makes the scanner less chatty, but does not hide warnings. Maybe there's no simple way to fix this since it's caused (I think) by the scanner not understanding some C syntax.

How about setting introspection to default off? At the moment the deluge of spurious scanner warnings is making genuine compiler warnings harder to spot.

Reply:
I've set it to false by default, we can revise this once the warnings are fixed :)